index.html

```diff
@@ -711,26 +712,22 @@ <h3>
           </li>
         </ol>
         <p data-dfn-for="PerformanceResourceTiming">
-          The <dfn>contentType</dfn> getter steps are to perform the following
-          steps:
+          The <dfn>contentType</dfn> getter steps are to return
+          <a>this</a>'s <a data-for="PerformanceResourceTiming">resource
+          info</a>'s [=response body info/content type=]'s <a data-cite=
+          "MIMESNIFF#mime-type-essence">essence</a>.
+        </p>
+        <p class='note'>
```

Comment:
I think this shouldn't be a note, but a normative part of the prose. Maybe "return this's resource info's content type's essence. The user agent may truncate the value in an [implementation defined](https://infra.spec.whatwg.org/#implementation-defined) way." ?

Reply:
Updated